docs/reference/ccr/use-cases.asciidoc

```diff
@@ -0,0 +1,121 @@
+[role="xpack"]
+[testenv="platinum"]
+[[ccr-use-cases]]
+=== {ccr-cap} use cases
+++++
+<titleabbrev>Use cases</titleabbrev>
+++++
+
+Use {ccr} to enable several use cases within the Elastic Stack:
+
+* <<ccr-disaster-recovery,Disaster recovery>> in case a primary cluster fails,
+with a secondary cluster serving as a hot backup
+* <<ccr-data-locality,Data locality>> to serve reads locally and reduce costly latency
+* <<ccr-centralized-reporting,Centralized reporting>> for preventing search load from interfering with indexing by offloading search to a secondary cluster
```

Comment:
"for minimizing network traffic and latency in querying multiple geo-distributed ES clusters or for preventing search load from interfering with indexing by offloading search to a secondary cluster"